packages/web/src/scss/components/Modal/_ModalBody.scss

```diff
@@ -9,20 +9,12 @@
     flex: 1 1 auto;
     min-height: map.get(theme.$typography, mobile, font-size) * map.get(theme.$typography, mobile, line-height); // 1.
     padding-inline: theme.$common-padding-x;
+    padding-block: theme.$body-padding-y;
```

Comment:
This is BC and we want to implement this as a separate issue with a feature flag. See DS-763.